src/main/java/org/recordrobotics/charger/control/DoubleControl.java

```diff
@@ -24,6 +24,21 @@ public double getDriveLat() {
 		return -_gamepad2.getLeftX();
 	}
 
+	@Override
+	public ClawState getClawTurn() {
+		// Trigger mimics button-like behavior
+		boolean cube = _gamepad2.getLeftTriggerAxis() > TRIGGER_THRESHOLD;
+		boolean cone = _gamepad2.getRightTriggerAxis() > TRIGGER_THRESHOLD;
+
+		// Cube takes precedence
+		if (cube) {
+			return ClawState.CUBE;
+		} else if (cone) {
+			return ClawState.CONE;
+		}
+		return ClawState.NEUTRAL;
+	}
```

Comment:
with this code, would i have to hold the trigger the entire time to close the claw, or could I just tap it and it would close by itself? which one is more efficient?

Reply:
We thought holding it down would be less confusing to drivers